C++/mim/test/PIMrunner/ionosphere_iri.h

```diff
@@ -32,7 +32,7 @@ namespace MIM_PIM {
 
 
 
-extern const char IRI_DATA_PATH_DEFAULT[] MIMSTR(INSTALLDIR"/libdata/IRI/");    
+extern const char IRI_DATA_PATH_DEFAULT[] MIMSTR(INSTALLDIR##"/libdata/IRI/");    
```

Comment:
@twillis449 I had to use concatenate here to make this compile ??? am I doing something stupid?

Reply:
nvm